content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -719,9 +719,13 @@ is turned on.
   more than one default is specified, the admission plugin forbids the creation of
   all PVCs.
 * If the admission plugin is turned off, there is no notion of a default
-  StorageClass. All PVCs that have no `storageClassName` can be bound only to PVs that
-  have no class. In this case, the PVCs that have no `storageClassName` are treated the
-  same way as PVCs that have their `storageClassName` set to `""`.
+  StorageClass. All PVCs that have `storageClassName` set to `""` can be
```

Comment:
Just to summarise the below two sentences and check if my understanding is right, do we mean to say that if the PVC has the `storageClassName`missing it can be later updated once the storage class becomes available but cannot be bound to a PV with the value `""`. Whereas if it's initially bound to `""` then it can only be bound to a PV with value `""` and if a storage class becomes available later we can't update the value?